lib/beaker/host/unix.rb

```diff
@@ -35,12 +35,14 @@ def determine_ssh_server
     end
 
     def external_copy_base
-      return @external_copy_base if @external_copy_base
-      @external_copy_base = '/root'
-      variant, version, arch, codename = self['platform'].to_array
-      # Solaris 10 uses / as the root user directory. Solaris 11 uses /root (like most).
-      @external_copy_base = '/' if variant == 'solaris' && version == '10'
-      @external_copy_base
+      @external_copy_base ||= begin
+        if self['platform'].variant == 'solaris' && self['platform'].version == '10'
```

Comment:
@ekohl I think this broke beaker-puppet:
https://github.com/voxpupuli/beaker-puppet/actions/runs/4261988674/jobs/7416955740#step:4:143

Reply:
Only the test suite, which used a mock object to pretend to be a platform. Fixed in https://github.com/voxpupuli/beaker-puppet/pull/209